hooks/basic/collector.py

```diff
@@ -538,26 +544,56 @@ def _get_item_type_from_settings(self, settings, path, is_sequence):
 
         # default values used if no specific type can be determined
         item_type = "file.unknown"
+        # work_path_template default value is none so it won't create a context for the item
+        work_path_template = None
 
         # keep track if a common type was identified for the extension
         common_type_found = False
 
-        # look for the extension in the common file type info dict
-        for item_type, type_info in settings["Item Types"].value.iteritems():
+        # mapping to track how many item types contain the given extension
+        template_item_type_mapping = dict()
 
+        # look for the extension in the common file type info dict
+        # using the raw value to get a raw work_path_template
+        for current_item_type, type_info in settings["Item Types"].raw_value.iteritems():
```

Comment:
I will have to query that again using `settings["Item Types"].raw_value.get(current_item_type)` since value is a dict and not a settings object.
Same will have to be applied in `_get_work_path_template_from_settings()` I'm thinking of updating `_get_item_type_info()` to have `get_raw_value=False` variable so that whoever needs raw values can ask for it explicitly.